target/linux/ipq40xx/files-4.19/arch/arm/boot/dts/qcom-ipq4018-jalapeno.dtsi

```diff
@@ -0,0 +1,269 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Please can you consider adding `SPDX-License-Identifier: GPL-2.0-or-later OR MIT` as in [DTS Checklist](https://openwrt.org/submitting-patches#dts_checklist) ?
> Having MIT as alternative is sometimes preferred by upstream maintainers
and allows sharing that simple code with other projects. We don't really
want multiple DTS versions for the same device.

Reply:
Yes, I added the OR MIT statement but did not push it yet.